src/components/admin/errors/BlankPage.tailwind.ts

```diff
@@ -0,0 +1,10 @@
+//   font-variation-settings: "wght" 350, "ital" 0, "SRFF" 20;
```

Comment:
[Question]  Do we still need it?

Reply:
Maybe we do, the issue I had was how to translate it to Tailwind, so instead I just left it there as a reminder. Maybe a more complete comment would have  been great.